docs/user/boot_options/index.md

```diff
@@ -38,6 +38,17 @@ first one can be removed anytime.
   inst.config_url=usb:///agama.yaml
   ```
 
+- `inst.dud`
+  Allows to patch or extend the Live Media by ponting to so called Driver Update. It currently
```

Comment:
```suggestion
  Allows to patch or extend the Live Media by pointing to so called Driver Update. It currently
```

Reply:
I've fixed this in a commit, sorry for being blind and not seeing you have created a commit suggestion.